dts/bindings/input/cirque,pinnacle-i2c.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Ilia Kharin
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cirque Pinnacle 1CA027 ASIC trackpad connected through I2C
+
+compatible: "cirque,pinnacle"
+
+include: ["spi-device.yaml", "cirque,pinnacle-common.yaml"]
```

Comment:
looks like you mixed up includes for spi-device.yaml and i2c-device.yaml

Reply:
Good catch! It should be fixed now. Interesting, it doesn't affect how it works.